package.json

```diff
@@ -1,85 +1,21 @@
 {
-  "name": "renovate-config-seek",
```

Comment:
I assume this means that we are moving from a versioned npm model (repos must explicitly upgrade to the new package version) to a GitHub default branch model (repos automatically receive merged changes). I like this change but is it contentious to anyone else?

Reply:
They're deprecated, so I hope not: https://docs.renovatebot.com/config-presets/#npm-hosted-presets